packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.controller.js

```diff
@@ -268,6 +277,16 @@ export default class PciTrainingJobsSubmitController {
     return this.PriceFormatter.format(flavor.catalog.price.value * 60);
   }
 
+  getFlavorPriceWithResources(flavor) {
+    if (!flavor.catalog) return 0;
+    return this.resourceN * flavor.catalog.priceInUcents * 60;
+  }
+
+  getFlavorPriceTaxWithResources(flavor) {
+    if (!flavor.catalog) return 0;
+    return this.resourceN * flavor.catalog.tax * 60;
```

Comment:
```suggestion
    if (!flavor.catalog) return 0;
    return this.resourceN * flavor.catalog.tax * 60;
```
return flavor.catalog ? this.resourceN * flavor.catalog.tax * 60 : 0;